### PR TITLE
login-sign-up

### PR DESCRIPTION
**Pull request by Mandhularajitha**

### CODE REVIEW

File: loginsignup.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+function setFormMessage(formElement, type, message) {
+    const messageElement = formElement.querySelector(".form__message");
+
+    messageElement.textContent = message;
+    messageElement.classList.remove("form_message--success", "form_message--error");
```
Comment (aniketxparihar) [inline loginsignup.js:5]: you have used just one _ here in form__message
Everything else looks clean

**Review by aniketxparihar — APPROVED** [review]

**Comment by samflab** [comment]
add a description and screenshots to your PR
and deploy your project on Netlify

File: loginsignup.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    background-size: cover;
+}
+
+.container {
+    width: 30rem;
+    margin: 5rem;
+    padding: 2rem;
+    box-shadow: 0 0 40px rgba(0, 0, 0, 2);
+    border-radius: var(--border-radius);
+    background: #ffffff;
```
Comment (samflab) [inline loginsignup.css:23]: add a css variable for this, don't use colours directly.

**Review by samflab — APPROVED** [review]

**Review by Sindhu983 — APPROVED** [review]

File: loginsignup.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    --color-primary-dark: #007f67;
+    --color-secondary: #252c6a;
+    --color-error: #cc3333;
+    --color-success: #4bb544;
+    --border-radius: 4px;
+
+
+    margin: 0;
+    height: 100vh;
+    font-size: 18px;
```
Comment (simran61) [inline loginsignup.css:13]: try using relative units for all values greater than 10px

File: loginsignup.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.form__input {
+    display: block;
+    width: 100%;
+    padding: 0.75rem;
+    box-sizing: border-box;
+    border: 1px solid #dddddd;
+    outline: none;
+    background: #eeeeee;
```
Comment (simran61) [inline loginsignup.css:78]: try to use css variables instead of using hex code directly

**Review by simran61 — APPROVED** [review]

**Pull request merged**